setup.cfg

```diff
@@ -17,10 +17,11 @@ packages = find:
 python_requires = >=3.6
 setup_requires = setuptools_scm
 install_requires =
-    astropy
+    astropy<4.3
```

Comment:
The trick is to use plain `astropy` here and then `astropy!=4.3` under `[docs]`

Reply:
Done